documentation/src/main/asciidoc/topics/proc_server_configuring_resp.adoc

```diff
@@ -105,3 +105,20 @@ include::json/resp_connector.json[]
 ----
 include::yaml/resp_connector.yaml[]
 ----
+
+[discrete]
+=== Clustered configuration
+
+{brandname} is a horizontally scalable Resp-compatible server with high availability capabilities.
+{brandname} provides clustering mechanisms with automatic failover detection and membership discovery by default.
+Topology changes are handled automatically, and adding or removing nodes to the cluster is a streamlined process that automatically redistributes entries between nodes without downtime.
+The cluster is homogeneous, where nodes are the primary and the backup owners of entries, guaranteeing that entries are available even in case of server failures.
+
+Backed by a consistent hash, compatible with the hash-slot algorithm in Resp clients, entries are distributed across cluster members.
+{brandname} automatically redirects requests to the proper entry owner.
+As a result, hash tags in keys and the `-MOVED` error response for requests are not required even for multi-key operations accessing keys with different owners.
+
+[TIP]
+====
+{brandname} allows multiple logical databases with the SELECT command in clustering mode.
```

Comment:
I wonder if we could fix the FLUSHALL command description within this PR as it's outdated and states that Infinspan doesn't support multiple databases

Reply:
Sounds good to me. I'll double-check all the commands and add anything missing as well.